Gemfile

```diff
@@ -0,0 +1,72 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.7.2'
+
+# Reduces boot times through caching; required in config/boot.rb
+gem 'bootsnap', '>= 1.4.4', require: false
+# to use new version of bootstrap https://github.com/bootstrap-ruby/bootstrap_form/tree/bootstrap-5
+# gem "bootstrap_form", git: "https://github.com/bootstrap-ruby/bootstrap_form.git", branch: "bootstrap-5"
+gem "bootstrap_form", github: "bootstrap-ruby/bootstrap_form", branch: "bootstrap-5"
+# https://github.com/thoughtbot/factory_bot_rails
+gem 'factory_bot_rails', '~> 6.1'
+# https://github.com/faker-ruby/faker
+gem 'faker', '~> 2.17'
+# Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
+gem 'jbuilder', '~> 2.7'
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails', branch: 'main'
+gem 'rails', '~> 6.1.3', '>= 6.1.3.1'
+# Use postgress as the database for Active Record
+gem 'pg'
+# Use Puma as the app server
+gem 'puma', '~> 5.0'
+# Use SCSS for stylesheets
+gem 'sass-rails', '>= 6'
+# SimpleCov is a code coverage https://github.com/simplecov-ruby/simplecov/
+gem "simplecov", "0.17"
+gem "simplecov-small-badge", "~> 0.2.4"
+# Turbolinks makes navigating your web application faster. Read more: https://github.com/turbolinks/turbolinks
+gem 'turbolinks', '~> 5'
+# Windows does not include zoneinfo files, so bundle the tzinfo-data gem
+gem 'tzinfo-data', platforms: %i[mingw mswin x64_mingw jruby]
+# Windows Directory Monitor (WDM) is a library which can be used to monitor directories for changes.
+gem 'wdm', '>= 0.1.0', platforms: %i[mingw mswin x64_mingw jruby]
+# Transpile app-like JavaScript. Read more: https://github.com/rails/webpacker
+gem 'webpacker', '~> 5.0'
+
+# Use Redis adapter to run Action Cable in production
```

Comment:
Are these gems going to be used later on?

Reply:
I don't Know , they were generated by rails , I think they are very common